mat64/dense.go

```diff
@@ -36,24 +36,26 @@ type Dense struct {
 	capRows, capCols int
 }
 
-// NewDense creates a new matrix of type Dense with dimensions r and c.
-// If the mat argument is nil, a new data slice is allocated.
+// NewDense creates a new Dense matrix with r rows and c columns. If
+// len(data) == r*c, data is used as the underlying slice (changes to the matrix
```

Comment:
"data is used as the backing slice. ... Changes to elements of the returned Dense will be reflected in data."

Panic warning in the elipsis.